controllers/ramenconfig_test.go

```diff
@@ -9,16 +9,33 @@ import (
 	. "github.com/onsi/gomega"
 	ramen "github.com/ramendr/ramen/api/v1alpha1"
 	"github.com/ramendr/ramen/controllers"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/yaml"
 )
 
 func configMapUpdate() {
 	ramenConfigYaml, err := yaml.Marshal(ramenConfig)
 	Expect(err).To(Succeed())
 
-	configMap.Data[controllers.ConfigMapRamenConfigKeyName] = string(ramenConfigYaml)
+	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		var err error
 
-	Expect(k8sClient.Update(context.TODO(), configMap)).To(Succeed())
+		err = k8sClient.Get(context.TODO(),
+			types.NamespacedName{Name: controllers.HubOperatorConfigMapName, Namespace: ramenNamespace},
+			configMap)
```

Comment:
This is accepted by the linter but is unreadable. When creating structs always
use one key: value pair per line style, unless the entire line is very short:

    types.NamespacedName{
        Name: controllers.HubOperatorConfigMapName,
        Namespace: ramenNamespace,
    },

Also this is better done once outside of the retry closure:

    configMapName := types.NamespacedName{
        Name: controllers.HubOperatorConfigMapName,
        Namespace: ramenNamespace,
    }

    retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
        err := k8sClient.Get(context.TODO(), configMapName, configMap)
        ...
    })

Reply:
Done